src/bbs2gh/Handlers/MigrateRepoCommandHandler.cs

```diff
@@ -136,7 +136,7 @@ private bool ShouldGenerateArchive(MigrateRepoCommandArgs args)
 
     private bool ShouldDownloadArchive(MigrateRepoCommandArgs args)
     {
-        return args.SshUser.HasValue() || args.SmbUser.HasValue();
+        return args.SshUser.HasValue() || args.SmbUser.HasValue() || args.SshHost.HasValue();
```

Comment:
No need to add the ssh host here, our indicators can still be the ssh user and smb user.